pkg/config/load_test.go

```diff
@@ -85,6 +85,49 @@ func TestExtractRepoElementsFromPath(t *testing.T) {
 	}
 }
 
+func TestInfo_IsComplete(t *testing.T) {
```

Comment:
50% of the code in `IsComplete()` is about forming the correct error message -- we should validate those

Reply:
OK